test/functional/wallet_labels.py

```diff
@@ -25,6 +25,41 @@ def set_test_params(self):
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
+    def invalid_label_name_test(self):
+        node = self.nodes[0]
+        address = node.getnewaddress()
+        pubkey = node.getaddressinfo(address)['pubkey']
+        rpc_calls = [
+            [node.getnewaddress],
+            [node.setlabel, address],
+            [node.getaddressesbylabel],
+            [node.importpubkey, pubkey],
+            [node.addmultisigaddress, 1, [pubkey]],
+            [node.getreceivedbylabel],
+        ]
+        if self.options.descriptors:
+            r = node.importdescriptors([{
```

Comment:
nit: would avoid using one-letter variables, the brevity isn't necessary here imo and it's just slightly more confusing
```suggestion
            response = node.importdescriptors([{
```